var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -54,3 +57,5 @@ def configure_args(self):
             '--enable-pc-files',
             '--with-pkg-config-libdir={0}/lib/pkgconfig'.format(self.prefix)
         ]
+        if '+symlinks' in self.spec:
+            opts += ["--enable-symlinks"]
```

Comment:
`opts` is never returned. I doubt this package actually works. Did you test this at all?